src/main/java/com/lorepo/icplayer/client/module/text/AudioButtonWidget.java

```diff
@@ -133,6 +133,17 @@ public String getId() {
         return null;
     }
 
+    @Override
+	public boolean getResetStatus() {
+        // 	not implement
```

Comment:
W nawiązaniu do naszej rozmowy należy zmienić nazwę komentarza na `TODO not implement`